base/utils/R/read.output.R

```diff
@@ -68,7 +68,7 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
   }
 
   years <- start.year:end.year
-  run_origin <- paste0(start.year, "-01-01")
+  run_origin <- paste0(start.year, "-01-01") #-- should we be hard-coding this assumption?
```

Comment:
This comment shouldn't be committed

Reply:
Yup, that was a temp note

Reply:
Sorry @infotroph, I made the changes but havent yet pushed due to the doc errors, I will not be merging until all changes made.  My mistake